### PR TITLE
Change out or upgrade most everything

### PR DESCRIPTION
**Pull request by mxmeinhold**
Change out the boilerplate config for newer recommendations, and upgrade dependencies along the way.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,18 +4,19 @@
...
+    "@testing-library/jest-dom": "^5.11.9",
+    "@testing-library/react": "^11.2.3",
+    "@testing-library/user-event": "^12.6.2",
     "csh-material-bootstrap": "^4.5.2",
     "history": "^5.0.0",
     "oidc-client": "^1.10.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
```
Comment (devinmatte) [inline package.json:15]: Why 16? 17 is out

**Comment by mxmeinhold** [comment]
A dependency of react-scripts has a peer dep of 14-16

**Comment by mxmeinhold** [comment]
Oh apparently replying to the email doesn't reply in thread. @devinmatte react-scripts has dependencies that have a peer dep of react 14-16.

**Comment by mxmeinhold** [comment]
Wait 2 months, and turns out they upgrade things. Look good to you now @matted?

**Pull request closed without merging**